plugins/module_utils/common.py

```diff
@@ -782,6 +782,42 @@ def create_project_request(self, definition):
         result['method'] = 'create'
         return result
 
+    def extract_selectors(self, instance):
```

Comment:
The method don't use `self`, can we convert it in a function and add test-coverage?